src/main/java/com/ksm/wstbackoffice/controller/AddressController.java

```diff
@@ -30,7 +30,14 @@ public AddressController(AddressService addressService) {
 
     @GetMapping("{id}")
     public ResponseEntity<Address> findById(@PathVariable long id) {
+        if (id < 0 || id == 0)
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
+
         Address address = addressService.findById(id);
+
+        if (address == null)
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body(null);
```

Comment:
Is it possible to omit body instead of passing null value?

Reply:
body statement  is mandatory